eng/scripts/Publish-Crates.ps1

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env pwsh
+
+#Requires -Version 7.0
+param(
+  [string]$PackagesPath,
+  [string[]]$CrateNames,
+  [string[]]$AdditionalOwners,
+  [string]$Token
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 2.0
+
+function TryAddOwners($crateName) {
+  foreach ($owner in $AdditionalOwners) {
+    Write-Host "Adding owner: '$owner' to crate: '$crateName'"
+    # https://doc.rust-lang.org/cargo/reference/registry-web-api.html#owners-add
+    # ignore errors is owner already exists
+    $body = @{ users = @($owner) } | ConvertTo-Json
+
+    $response = Invoke-WebRequest -Method Put -Uri "https://crates.io/api/v1/crates/$crateName/owners" `
+      -Headers @{ Accept = 'application/json'; Authorization = $Token } `
+      -ContentType 'application/json' `
+      -Body $body`
+      -SkipHttpErrorCheck
+
+    if ($response.StatusCode -ge 400 -and $response.Content -notmatch 'already an owner') {
```

Comment:
Nothing more programmatic than `already an owner`? Seems a bit fragile, but we can always fix it later if it changes. But can we restart publishing in such an event safely enough?

Reply:
It returns a 400 status code with no error specific headers and the response body:
```json
{"errors":[{"detail":"`hallipr` is already an owner"}]}
```

Reply:
It'll have to do.